ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -1340,5 +1340,12 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
```

Comment:
just for info, now access control for same event can be  grouped like
`{
    "LiveFrom": "01/01/2017",
    "CaseTypeID": "CARE_SUPERVISION_EPO",
    "CaseEventID": "manageRepresentatives",
    "AccessControl": [
      {
        "UserRoles": ["A","B"],
        "CRUD": "CRU"
      },
      {
        "UserRoles": ["C"],
        "CRUD": "R"
      }
    ]
  }`